core/frontend/src/store/bridget.ts

```diff
@@ -43,6 +45,16 @@ class BridgetStore extends VuexModule {
     this.available_serial_ports = available_serial_ports
     this.updating_bridges = false
   }
+
+  @Mutation
+  startFetching(): void {
```

Comment:
If we are aiming to a generic approach, I would recommend to have this kind of call as a register method, where multiple components can "enable"/"disable" the fetch, and after all components get dismounted the register will hit zero and should_fetch will be false.